### DISCUSSION TITLE
无法获取百度网盘密钥

### DISCUSSION THREAD

**Discussion by fluenceeee** [AlistGo/alist#4956] [Q&A]
您遇到了错误！
错误代码：redirect_uri_mismatch
错误信息：Invalid redirect uri
详细描述：授权回调页地址错误，具体内容请开发者参考[“redirect uri”验证方式](http://developer.baidu.com/wiki/index.php?title=docs/oauth/redirect)
麻烦您告诉我这是哪来的问题
![Snipaste_2023-08-08_13-09-30](https://github.com/alist-org/alist/assets/141704770/d3a4e665-3884-4646-b241-07ca36cbe80a)

**Comment by mingtianshi** [comment]
Cookie扫二维码应该就能用，刷新一下网页重试看看

**Comment by anwen-anyi** [comment]
换 V3文档里面的，https://alist.nn.ci/zh/guide/drivers/baidu.html
我觉得你还在使用V2的文档...........

**Comment by fluenceeee** [comment]
我是小白 看不太懂 咱们有群聊或者教程么


&nbsp;


你说你喜欢大海我就超浪
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年8月8日(星期二) 下午4:31
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [alist-org/alist] 无法获取百度网盘密钥 (Discussion #4956)





 
换 V3文档里面的，https://alist.nn.ci/zh/guide/drivers/baidu.html
 我觉得你还在使用V2的文档...........
